homework13/SortClass.java

```diff
@@ -0,0 +1,38 @@
+package homework13;
+
+import java.util.Arrays;
+
+public class SortClass {
+
+    public static int[] bubbleArray(int[] array) {
+        for (int i = 0; i < array.length; i++) {
+            for (int j = 0; j < array.length - i - 1; j++) {
+                if (array[j] > array[j + 1]) {
+                    int temp = array[j];
+                    array[j] = array[j + 1];
+                    array[j + 1] = temp;
+                }
+            }
+        }
+        return array;
+    }
+
+    public static int[] insertion(int[] array) {
+        for (int i = 0; i < array.length; i++) {

```

Comment:
It would be better to change `int i = 0` to `int i = 1` as the first element is sorted by default, so you should pick up the next one.